datalad/distribution/create.py

```diff
@@ -86,14 +85,6 @@ class Create(Interface):
             doc="""enforce creation of a dataset in a non-empty directory""",
             action='store_true'),
         description=dataset_description,
-        add_to_super=add_to_superdataset,
```

Comment:
Thinking about it again, it may make most sense to just move the add-to-super logic into the crawler code. If there is not superdataset instance anywhere at this point, we have to discover it anyway.